src/definitions/schema.d.ts

```diff
@@ -1449,7 +1449,7 @@ export const enum GQLTransactionCurrency {
 }
 
 export const enum GQLPriceBillingCycle {
-  month = 'month',
+  monthly = 'monthly',
```

Comment:
FYI, Stripe use noun such as `month` and `day`, so I use `month` here instead of `monthly`.